components/Buttons/src/MDCButton.m

```diff
@@ -102,6 +102,7 @@ @interface MDCButton () {
   // Cached accessibility settings.
   NSMutableDictionary<NSNumber *, NSString *> *_accessibilityLabelForState;
   NSString *_accessibilityLabelExplicitValue;
+  BOOL _mdc_adjustsFontForContentSizeCategory;
```

Comment:
lets add an extra spaced since it is not related to the cache accessibility settings.